app/controllers/provider_foreman_controller.rb

```diff
@@ -63,11 +63,21 @@ def new
     assert_privileges("provider_foreman_add_provider")
     @provider_cfgmgmt = ManageIQ::Providers::ConfigurationManager.new
     @provider_types = ["Ansible Tower", ui_lookup(:ui_title => 'foreman')]
+    @server_zones = []
```

Comment:
Also, we probably only want to display zones in the current region.  You can limit that with a minor change:
```ruby
@server_zones = Zone.in_my_region.order('lower(description)').pluck(:description, :name)
=> [["Default Zone", "default"], ["nothing to see here", "empty"]]
```